index.html

```diff
@@ -5,43 +5,187 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
-  <title>Document</title>
+  <title>Women in Tech</title>
 </head>
 <body>
 
-  <!-- Header Starts Here -->
-    <header>
-      <!-- Top Section -->
-      <div class="top-section">
-        <ul class="top-section-ul">
-          <li class="top-section-list">facebook</li>
-          <li class="top-section-list">facebook</li>
-          <li class="top-section-list">facebook</li>
-          <li class="top-section-list">facebook</li>
+ <main>
+   <!-- Header Starts Here -->
+   <header>
+    <!-- Top Section -->
+    <div class="top-section">
+      <ul class="top-section-ul">
+        <li class="top-section-list"><img src="Assets\icons8-facebook-18.png" alt="facebook"/></li>
+        <li class="top-section-list"><img src="Assets\icons8-twitter-18.png" alt="twitter"/></li>
+        <li class="top-section-list">English</li>
+        <li class="top-section-list">My Page</li>
+        <li class="top-section-list">Logout</li>
+      </ul>
+    </div>
+
+    <!-- Navbar -->
+    <!--Mobile-->
+    <nav class="navbar-mobile">
+      <div class="menu">
+        <a> <img src="Assets\mobile_menu 1.png" class="menubar" alt="navbar"> </a>
+         </div>
+    </nav>
+
+    <nav class="mobile-menu">
+      <ul class="mobile-menu-list">
+       <img src="Assets\icons8-close-24 (6).png" class="closebtn" alt="close">
+       <li class="menu-list"> <a href="about.html" class="menu-link">About</a></li>
+     <li class="menu-list"> <a href="#about" class="menu-link">Program</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">Join</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">Sponsor</a></li>
+     <li class="menu-list"> <a href="#contact-me" class="menu-link">News</a></li>
+    </ul>
+     </nav>
+    
+      <!-- Nav Items -->
+      <nav class="navbar-container">
+        <a href="index.html" class="p-logo"><p class="logo">WIT</p></a>
+        <ul class="navbar-items">
+          <li>
+            <a href="about.html" class="menu-link-desktop">About</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Program</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Join</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">Sponsor</a>
+          </li>
+
+          <li>
+            <a href="#" class="menu-link-desktop">News</a>
+          </li>
+
+          <li class="list-items">
+            <a href="#">WITMP 2023</a>
+          </li>
         </ul>
-      </div>
+      </nav>
+  </header>
+<!-- Header Ends Here -->
+
+<!-- Main Starts Here -->
+<section class="theme-info">
+  <!-- Hero -->
+  <div class="first-section">
+  <div>
+  <h4 class="theme-intro">"Great News!"</h4>
+  <h2 class="theme">WOMEN IN TECH <br> MENTORSHIP PROGRAMME <br> 2023.</h2>
```

Comment:
- [ ] kindly use an h1 tag here, as it is your main heading.
- [ ] please kindly make the heading bigger, bolder like in the original design. 
- [ ] kindly add margin left and right to all sections, so it matches the design.
yours
![Screenshot (1163)_LI](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/922e027c-2fe0-4876-a82b-29ee7334e796)
original
![Screenshot (1164)](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/65298048/a9c8fac2-7c0e-4ae9-ac97-3da89e331d28)

Reply:
Thank you so much for your review, I have fixed the padding and margin for this screen size.
![hero-section](https://github.com/JoyTaribagshaw/module-1-capstone-project/assets/111312788/bb4b9820-e70b-4219-89b8-d4ac52288adc)